### PR TITLE
Added Employee Manager.

### PR DESCRIPTION
**Pull request by TheStigger**

### CODE REVIEW

**Comment by Fedorof** [comment]
Tests show this:

    ERROR: carcassonne.helpers.sse.tests.test_employee (unittest.loader._FailedTest)
    ----------------------------------------------------------------------
    ImportError: Failed to import test module: carcassonne.helpers.sse.tests.test_employee
    Traceback (most recent call last):
      File "/home/ubuntu/.pyenv/versions/3.5.0/lib/python3.5/unittest/loader.py", line 428, in _find_test_path
        module = self._get_module_from_name(name)
      File "/home/ubuntu/.pyenv/versions/3.5.0/lib/python3.5/unittest/loader.py", line 369, in _get_module_from_name
        __import__(name)
      File "/home/ubuntu/workspace/SSP/carcassonne/helpers/sse/tests/test_employee.py", line 3, in <module>
        from carcassonne.helpers.sse import list_to_employee
    ImportError: cannot import name 'list_to_employee'

File: SSP/carcassonne/helpers/sse/converters.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
+"""
+Collection of functions for working with Employee objects.
+
+Warning: these functions are to be converted to Employee methods. Do not
+use them outside the package of Employee class.
+
+"""
+
+from carcassonne.helpers.sse.employee import Employee, Workplace
```
Comment (Fedorof) [inline SSP/carcassonne/helpers/sse/converters.py]: I would add all helpful stuff to `helpers/sse/__init__.py` to make import easier.

File: SSP/carcassonne/helpers/sse/converters.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        ["id_sse", 'vertical', 'project', 'position', 'name',
+         'table', 'room', 'location']
+
+    :raise: WrongParameterError list_employee is not
+    iterable of exactly 8 elements
+
+    """
+    try:
+        id_sse, vertical, project, position, name, table, room, location \
+            = list_employee
```
Comment (Fedorof) [inline SSP/carcassonne/helpers/sse/converters.py]: You should've left `=` on the same line

File: SSP/carcassonne/helpers/sse/manager.py [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+        If id is None, then assign new id. If id already specified, then it
+        should be unique.
+        :param new_employee: object of new employee
+        :type new_employee: _employee_cls.
+        :raise EmployeeIdError, ValueError: raises error in case of wrong id.
+        """
+        if new_employee.id_sse is None:
+            new_employee.id_sse = max(int(e.id_sse) for e in self._pool) + 1
+        elif not type(new_employee.id_sse) is str:
+            raise EmployeeIdError("Employee id type should be str.")
```
Comment (Fedorof) [inline SSP/carcassonne/helpers/sse/manager.py]: Are you sure it should be a string?

File: SSP/carcassonne/helpers/sse/tests/test_manager_dummy.py [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+        Test automatic creation of test_db.csv.
+        """
+        self.assertTrue(os.path.isfile(settings.DUMMY_CSV_PATH))
+
+    def test_db_copying(self):
+        """
+        Test copying dump.csv file content to test_db.csv.
+        """
+        self.assertTrue(
+            filecmp.cmp(settings.DUMMY_CSV_PATH, settings.DUMP_PATH))
```
Comment (Fedorof) [inline SSP/carcassonne/helpers/sse/tests/test_manager_dummy.py]: It's better to use `self.assertTrue` with a custom message.

File: SSP/carcassonne/helpers/sse/converters.py (hunk 1 above)
Comment (TheStigger) [inline SSP/carcassonne/helpers/sse/converters.py]: @Fedorof can we use relative imports instead of extracting imports to `__init__.py`?

Comment (Fedorof) [inline SSP/carcassonne/helpers/sse/converters.py]: Just add to `helpers/sse/__init__.py` these lines:

    from .employee import Employee, Workplace
    from .manager import get_manager, EmployeeIdError, ANY

Then update imports in modules and run tests to make sure there are no cyclic import dependencies.

**Comment by TheStigger** [comment]
/CC @Fedorof @ITsvetkoFF 
I've fixed all errors and remarks. Hope you'll approve this pull request :)

File: SSP/carcassonne/helpers/sse/__init__.py [hunk 5]
```diff
@@ -0,0 +1,3 @@
+from .employee import Employee, Workplace
+from .converters import list_to_employee, employee_to_list, WrongParameterError
```
Comment (Fedorof) [inline SSP/carcassonne/helpers/sse/__init__.py]: list_to_employee, employee_to_list should not be available outside the package (see the `converters`' module docsring) Remove them from here, please

File: SSP/carcassonne/helpers/sse/tests/test_converters.py [hunk 6]
```diff
@@ -0,0 +1,84 @@
+import doctest
```
Comment (Fedorof) [inline SSP/carcassonne/helpers/sse/tests/test_converters.py]: Just realized. A better place for tests is `SSP/carcassonne/tests/helpers/sse/`

File: SSP/carcassonne/helpers/sse/tests/test_manager_dummy.py [hunk 7]
```diff
@@ -0,0 +1,198 @@
+import os
+import filecmp
+from os.path import join
+
+from django.test import override_settings, TestCase
+
+from django.conf import settings
+
+from .. import Employee, Workplace, get_manager, EmployeeIdError
```
Comment (Fedorof) [inline SSP/carcassonne/helpers/sse/tests/test_manager_dummy.py]: When tests are moved to SSP/carcassonne/**tests**/helpers/sse/test_manager_dummy.py, use absolute import here

**Comment by TheStigger** [comment]
/CC @Fedorof @ITsvetkoFF 
Again fixed all remarks :)

File: SSP/carcassonne/tests/test_converters.py [hunk 8]
```diff
@@ -0,0 +1,85 @@
+import doctest
```
Comment (Fedorof) [inline SSP/carcassonne/tests/test_converters.py]: Again, the path should be `SSP/carcassonne/tests/helpers/sse/<test_file>`. Please, be more attentive.

File: SSP/carcassonne/tests/test_employee.py [hunk 9]
```diff
@@ -0,0 +1,43 @@
+import unittest
```
Comment (Fedorof) [inline SSP/carcassonne/tests/test_employee.py]: `SSP/carcassonne/tests/helpers/sse/<test_file>`.

File: SSP/carcassonne/tests/test_manager_dummy.py [hunk 10]
```diff
@@ -0,0 +1,199 @@
+import os
```
Comment (Fedorof) [inline SSP/carcassonne/tests/test_manager_dummy.py]: `SSP/carcassonne/tests/helpers/sse/<test_file>`.

File: SSP/carcassonne/helpers/sse/manager.py [hunk 11]
```diff
@@ -0,0 +1,226 @@
...
+
+        If id is None, then assign new id. If id already specified, then it
+        should be unique.
+        :param new_employee: object of new employee
+        :type new_employee: _employee_cls.
+        :raise EmployeeIdError, ValueError: raises error in case of wrong id.
+        """
+        if new_employee.id_sse is None:
+            new_employee.id_sse = max(int(e.id_sse) for e in self._pool) + 1
+        elif not type(new_employee.id_sse) is int:
```
Comment (Fedorof) [inline SSP/carcassonne/helpers/sse/manager.py]: `elif not isinstance(new_employee.id_sse, int):`

**Pull request merged**